src/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj

```diff
@@ -0,0 +1,237 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Microsoft.IO.Redist</AssemblyName>
+    <IsPartialFacadeAssembly>false</IsPartialFacadeAssembly>
+    <ProjectGuid>{FEF5811F-ED50-4407-A6B9-885EBD3206FB}</ProjectGuid>
+    <Configurations>netfx-Debug;netfx-Release</Configurations>
+    <DefineConstants>$(DefineConstants);MS_IO_REDIST</DefineConstants>
+    <LangVersion>7.3</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ClsCompliant>false</ClsCompliant>
+    <UWPCompatible Condition="'$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot'">true</UWPCompatible>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(BuildingInsideVisualStudio)' == 'true' or ('$(TargetsWindows)' == 'true' and '$(UWPCompatible)' != 'true')">
```

Comment:
What's the deal with this condition?

Reply:
It won't actually load the files in the Solution explorer without it.

Reply:
What if you remove the condition entirely.  I'm guessing you needed it back when you had a not supported assembly and the default configuration VS loaded was a not-supported config.